open_spiel/games/chess.h

```diff
@@ -242,6 +242,8 @@ class ChessGame : public Game {
     return chess::ObservationTensorShape();
   }
   int MaxGameLength() const override { return chess::MaxGameLength(); }
+  // There aren't chance nodes in this game.
```

Comment:
Let's put this in in the base Game - if type is deterministic, return 0 for MaxChanceNodes